src/shared/mkfs-util.c

```diff
@@ -361,6 +361,7 @@ int make_filesystem(
                                 "-m", "0",
                                 "-E", discard ? "discard,lazy_itable_init=1" : "nodiscard,lazy_itable_init=1",
                                 "-b", "4096",
+                                "-T", "default",
```

Comment:
btw, where is "default" actually define? it is mentioned in mke2fs man page, but it doesn't show up in my /etc/mke2fs.conf file.

(from the mke2fs man page this sounds like the right thing to invoke though)

Reply:
i mean, taking one step back: my previous thinking is that the mke2fs.conf nonsense is something we'd not want to rely on, hence we'd rather explicitly select all options, than to rely on that file.

I don't understand what "-T default" actually means though. Is that documented anywhere?

what actually happens if I explicitly specify "-I 256" *and* "-T default"? which one wins? I hope it's not the last specified argument? if so, we should probably switch the order here.

Reply:
so, hmm, if i look at mke2fs.conf the different profiles differ in blocksize, inode size, and "inode ratio".

The former two we set explicitly anyway. So wouldn't it be smarter to hardcode the inode ration on the cmdline too? That way we become independent of that config file, and behave the same everywhere.

or did you have anything spefifically in mind with "-T default" beyond these three settings?

Reply:
By using `-T default`, we default to whatever is configured in `base_features` in /etc/mke2fs.conf`. I would prefer to rely on that instead of setting default options ourselves because then we're responsible for making sure the defaults work everywhere. I'd rather leave that to the mkfs tool for now until we have a proper way to configure individual options in repart's config file.